src/page/Home.jsx

```diff
@@ -9,14 +9,16 @@ import {
 } from '../models/roomModel';
 import { getDbUser } from '../models/userModel';
 import RoomCard from '../components/RoomCard/RoomCard';
-import NewRoomModal from '../components/dashboard/NewRoomModal';
-import EditRoomModal from '../components/dashboard/EditRoomModal';
+import NewRoomModal from '../components/GeneralRoomModel/NewRoomModal';
+import EditRoomModal from '../components/GeneralRoomModel/EditRoomModal';
```

Comment:
just to check, are we going with `Model` for the folder name?

Reply:
oh shucks need to change it to modal! sorry!